### PR TITLE
Customize

### PR DESCRIPTION
**Pull request by dquartino**
Hi, here is a proposal to add a command to display a tag name on a detached branch.  I also added the possibility to define the user and the host to display in user@host.

Finally I added a new theme "neon".
image

I hope you find these additions useful !

### CODE REVIEW

**Comment by diogocavilha** [comment]
@all-contributors please add @dquartino  for code

File: app_config_sample [hunk 1]
```diff
@@ -9,3 +9,6 @@ time_format:%H:%M:%S
 separator:
 ps2:➜
 fresh_file
+user_name:
+host_name:
+show_tag_name:true
```
Comment (diogocavilha) [inline app_config_sample:14]: Everything is ok with the new keys:values. 
The only change needed here is that the last line should be the "fresh_file" string.

File: commands-handler.sh [hunk 2]
```diff
@@ -37,6 +37,14 @@ case "$1" in
     "--disable-double-line") fancygit_config_save "double_line" "false";;
     "--enable-rich-notification") fancygit_config_save "show_rich_notification" "true";;
     "--disable-rich-notification") fancygit_config_save "show_rich_notification" "false";;
+    "--enable-tag-name") fancygit_config_save "show_tag_name" "true";;
+    "--disable-tag-name") fancygit_config_save "show_tag_name" "false";;
```
Comment (diogocavilha) [inline commands-handler.sh:41]: I'm not sure if disabling/enabling tag name is interesting here. I mean, maybe wee need the tag name to be always visible, once we checkout to it. What do you think of it?

File: themes/default.sh [hunk 3]
```diff
@@ -93,6 +102,9 @@ fancygit_theme_builder() {
         prompt_path="${path}${prompt_env} ${prompt_path} ${path_end}${workdir_color_tag}${separator}${none}"
         PS1="${prompt_time}${prompt_user}${prompt_symbol}${prompt_path}${prompt_double_line} "
         return
+    elif [ "HEAD" = "$branch_name" ] && [ "true" = "$show_tag_name" ]
```
Comment (diogocavilha) [inline themes/default.sh:105]: You can replace the `elif` by an `if`, as it returns one line above.

**Review by diogocavilha — CHANGES_REQUESTED** [review]
@dquartino 

Thank you so much for the job you did here. I really appreciate that!

Good job!

Despite the comments I did requesting a few adjustments, I'd ask you just to change two more files: 

**version.sh**
The current Fancygit version is 7.0.0, you can change it to 7.1.0

**CHANGELOG.md**
Please, update the changelog file by adding some topics containing briefs of the changes you did. As you can see, there are plenty of entries there, so it's quite simple to create yours by following some example.

File: themes/default.sh (hunk 3 above)
Comment (dquartino) [reply themes/default.sh]: I didn't pay attention! It's ok !

File: app_config_sample (hunk 1 above)
Comment (dquartino) [reply app_config_sample]: Done !

**Comment by dquartino** [comment]
I finally removed the option to enable/disable the tag name.
You're right, it's better to have the tag name always visible.

Everything else is ok !
I made a little update on the neon theme too.

**Comment by diogocavilha** [comment]
Great job, @dquartino !!!

Thank you once again.

**Review by diogocavilha — COMMENTED** [review]
Great job!

**Pull request merged**